app/src/main/java/com/macbitsgoa/ard/models/User.java

```diff
@@ -0,0 +1,60 @@
+package com.macbitsgoa.ard.models;
+
+import com.google.firebase.auth.FirebaseUser;
+
+import com.macbitsgoa.ard.utils.AHC;
+
+/**
+ * Model class 'User'.
+ * This data is stored in firebase directory {@link AHC#FDR_USERS} with uid
+ * (obtained by {@link FirebaseUser#getUid()}) as key.
+ * A user has read access to data of all users, and write access to his/her info.
+ * @author Rushikesh Jogdand
+ */
+
+public class User {
+
+    /**
+     * User name (full name initially obtained from {@link FirebaseUser#getDisplayName()}).
+     */
+    private String name;
+
+    /**
+     * Email Id from {@link FirebaseUser#getEmail()}.
+     */
+    private String email;
+
+    /**
+     * Photo url from {@link FirebaseUser#getPhotoUrl()}.
+     */
+    private String photoUrl;
+
+    @SuppressWarnings("unused")
+    public User() {
+        // Default constructor necessary for firebase db.
```

Comment:
I don't suppose removing this will have any effect?
compiler produces empty constructor on its own anyways..